Rakefile

```diff
@@ -100,3 +100,35 @@ namespace :assets do
     FileUtils.rm_rf("#{AppPrototype::Container.config.root}/public/assets")
   end
 end
+
+task :routes do
+  require "roda"
+  @app = Class.new(Roda)
+
+  # get a list of route files for all our apps
+  app_paths = Pathname(__FILE__).join("../apps").realpath.join("*")
+  routes = Dir.glob("#{app_paths}/web/routes/*.rb").join(" ")
+  system("roda-parse_routes -f routes.json #{routes}")
+
+  # load the route_list plugin
+  @app.plugin :route_list
+  routes = @app.route_list
+
+  # determine the max lengths of our output strings so we can display a nice list of routes
```

Comment:
This part (including the width method) is _inspired_ by `ActionDispatch:: Routing:: ConsoleFormatter ` and I wonder if it's worth moving this out to something like that, though I doubt this code will need to be reused in a different context yet.